### ISSUE TITLE
Time picker not working

### ISSUE THREAD

**Issue by pupeno** [react-toolbox/react-toolbox#1264]
I'm experimenting with the time-picker and it's crashing. I'm creating one with a label, a value that's just a Date  and an on-change callback that prints the change to the console. You can see the results here:

http://reagent-toolbox-docs.dashman.tech/time-picker

Any ideas what's going on?

**Comment by jcmcneal** [comment]
Time Picker and Date Picker both crash for me like that. It loads and opens once but then spits out this error:

```
addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
```

Usually I've found this means React is being loaded more than once. This doesn't happen with Tabs, Dialog, Slider, and Progress Bar.

**Comment by javivelasco** [comment]
@pupeno did you get any progress on this? That error also happens when you pass a ref to an stateless component I think.

**Comment by pupeno** [comment]
No, not yet. I haven't looked a lot into this yet to be honest.
